ThunderCatsTests/UIAlertControllerTests.m

```diff
@@ -0,0 +1,66 @@
+//
+//  UIAlertControllerTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-09.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UIAlertController+TCAdditions.h"
+
+@interface UIAlertControllerTests : XCTestCase
+
+@end
+
+@implementation UIAlertControllerTests
+
+- (void)testNotImplementedAlert
+{
+    UIAlertController *alert = [UIAlertController tc_notImplementedAlert];
+    
+    XCTAssertEqualObjects(alert.title, @"Not Implemented");
+    XCTAssertEqualObjects(alert.message, @"This feature is not yet implemented.");
+    XCTAssertEqualObjects(alert.actions[0].title, @"Ok");
+}
+
+- (void)testAlertWithTitle
+{
+    NSString *title = @"Title";
+    NSString *message = @"Message Text";
+    NSString *cancelButtonTitle = @"Cancel";
+    
+    UIAlertController *alert = [UIAlertController tc_alertWithTitle:title message:message cancelButtonTitle:cancelButtonTitle cancelAction:nil];
+    
+    XCTAssertEqual(alert.title, title);
+    XCTAssertEqual(alert.message, message);
+    XCTAssertEqual(alert.actions[0].title, cancelButtonTitle);
```

Comment:
We should possibly write this as `alert.actions.firstObject.title` so that it doesn't crash if the actions array is empty for some reason.  We can also add an assertion to verify that the actions array contains exactly one item.  If for some reason this method returned an alert controller with two actions, this test would still pass, but the code would obviously be wrong.